core/src/main/java/com/adobe/aem/commons/assetshare/content/renditions/download/async/impl/AsyncAssetRenditionsDownloadServlet.java

```diff
@@ -65,7 +64,7 @@
     private static final String REQ_KEY_RENDITION_NAMES = "renditionName";
 
     private static final String PN_ALLOWED_RENDITION_NAMES = "allowedRenditionNames";
-    private static final String PN_BASE_ARCHIVE_NAME = "zipFileName";
+    private static final String PN_BASE_ARCHIVE_NAME = "fileName";
```

Comment:
@davidjgonzalez not quite related, but noticed this...

Reply:
Ill make a note to check which is right  - we can leave this one for now.